src/index.html

```diff
@@ -5,9 +5,50 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="product-card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="Product Image"
+        class="product-card__img"
+      >
+      <p class="product-card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+
+      <article class="product-card__review">
+        <div class="product-card__review-stars">
+          <div class="product-card__review-star-active"></div>
+          <div class="product-card__review-star-active"></div>
+          <div class="product-card__review-star-active"></div>
+          <div class="product-card__review-star-active"></div>
+          <div class="product-card__review-star"></div>
+        </div>
```

Comment:
Just use stars from previous task.
Checklist:
4. [TASK] - Make sure to reuse part of [Stars task](https://github.com/mate-academy/layout_stars) here, and keep it operational in case you change block modifier.